tools/scripts/generate_simplified_db_schema.py

```diff
@@ -0,0 +1,27 @@
+import re
+
+
+# Uncomment one of the imports depenging on the db that you want to generate simplified schema for
+from rotkehlchen.db.schema import DB_SCRIPT_CREATE_TABLES
+# from rotkehlchen.globaldb.schema import DB_SCRIPT_CREATE_TABLES
+
+
+# Replacing new lines and white spaces since they may vary if by an accident code of a
+# db upgrade was a bit different from the one that creates new tables
+create_script_from_constant = DB_SCRIPT_CREATE_TABLES.replace(' ', '').replace('\n', '').replace('\'', '"')  # noqa: E501
+
+# Make dictionary of table_name: table_properites. Without whitespaces. For example:
+#   "ens_mappings": "CREATETABLEIFNOTEXISTSens_mappings(addressTEXTNOTNULLPRIMARYKEY,ens_nameTEXTUNIQUE,last_updateINTEGERNOTNULL);"  # noqa: E501
+#
+regexp_result = re.findall(
+    pattern=r'CREATETABLEIFNOTEXISTS(.+?)\((.+?)\);',
+    string=create_script_from_constant,
+)
+# print it this way for easier copy-pasting
+print('{')
```

Comment:
Why print and copy paste? That's silly to do from a script.

As I said in the other parts of the review those dicts should go in their own file.
That file can be created by this script. And it would also be good to have the timestamp of generation at the very top to easily see when was that version generated, at what rotki version and by whom.